### PR TITLE
[Translation] Virtual DOM and Internals page in FAQ to Hindi 🇮🇳

### PR DESCRIPTION
**Pull request by bhatvikrant**
Translating the Virtual DOM and Internals page to Hindi.
Related to #1

- [x] वास्तविक DOM क्या है?

- [x] क्या शैडो DOM वास्तविक DOM के समान है?

- [x] “React फाइबर” क्या है?

### CODE REVIEW

**Comment by bhatvikrant** [comment]
Hi @saranshkataria, can I get an idea of when the PR will be reviewed tentatively if possible?

Thanks for your awesome efforts for maintaining this repo  😀

**Comment by saranshkataria** [comment]
@arshadkazmi42 will start with the first set of reviews, hopefully soon.

File: content/docs/faq-internals.md [hunk 1]
```diff
@@ -1,23 +1,23 @@
 ---
 id: faq-internals
-title: Virtual DOM and Internals
+title: वास्तविक DOM और इंटरनल्स
```
Comment (arshadkazmi42) [inline content/docs/faq-internals.md:3]: `वास्तविक` => `वर्चुअल`

change everywhere

File: content/docs/faq-internals.md [hunk 2]
```diff
@@ -1,23 +1,23 @@
...
+### वास्तविक DOM क्या है? {#what-is-the-virtual-dom}
 
-The virtual DOM (VDOM) is a programming concept where an ideal, or "virtual", representation of a UI is kept in memory and synced with the "real" DOM by a library such as ReactDOM. This process is called [reconciliation](/docs/reconciliation.html).
+वास्तविक DOM (VDOM) एक प्रोग्रामिंग कांसेप्ट है जहां एक आदर्श, या "वास्तविक", UI का रिप्रजेंटेशन मेमोरी में रखा जाता है और ReactDOM जैसी लाइब्रेरी द्वारा "वास्तविक" DOM के साथ सिंक में रखा जाता है। इस प्रक्रिया को [reconciliation](/docs/reconciliation.html) कहते हैं।
 
-This approach enables the declarative API of React: You tell React what state you want the UI to be in, and it makes sure the DOM matches that state. This abstracts out the attribute manipulation, event handling, and manual DOM updating that you would otherwise have to use to build your app.
+यह तरीका React कि वर्णनात्मक API को सक्षम बनाता है: आप React को यह बताएं कि आप किस state में UI को चाहते हैं और यह सुनिश्चित करता है कि DOM उस अवस्था से मेल खाता है। यह एट्रिब्यूट मैनीपुलेशन, ईवेंट हैंडलिंग और मैन्युअल DOM अपडेट को अमूर्त करता है वरना जिसका आपको अपने ऐप को बनाने के लिए उपयोग करना पढता।
```
Comment (arshadkazmi42) [inline content/docs/faq-internals.md:13]: `वर्णनात्मक` => `डेक्लेरेटिव`

any suggestions? `वर्णनात्मक` sounds complex

Comment (arshadkazmi42) [inline content/docs/faq-internals.md:13]: `आप React को यह बताएं कि आप किस state में UI को चाहते हैं और यह सुनिश्चित करता है कि DOM उस अवस्था से मेल खाता है।`

can you revisit this translation. sounds odd.

Comment (arshadkazmi42) [inline content/docs/faq-internals.md:13]: `अमूर्त` => `अब्स्ट्रेक्ट्स`

File: content/docs/faq-internals.md [hunk 3]
```diff
@@ -1,23 +1,23 @@
...
+### क्या शैडो DOM वास्तविक DOM के समान है? {#is-the-shadow-dom-the-same-as-the-virtual-dom}
 
-No, they are different. The Shadow DOM is a browser technology designed primarily for scoping variables and CSS in web components. The virtual DOM is a concept implemented by libraries in JavaScript on top of browser APIs.
+नहीं, वे अलग हैं। शैडो डोम एक ब्राउज़र तकनीक है जिसे मुख्य रूप से वेरिएबल्स को खोजने और web कौम्पोनॅन्टस में CSS के लिए डिज़ाइन किया गया है। वास्तविक DOM ब्राउज़र APIs के ऊपर, जावास्क्रिप्ट लाइब्रेरीज द्वारा इम्प्लीमेंट किया गया एक कांसेप्ट है।
 
-### What is "React Fiber"? {#what-is-react-fiber}
+### "React फाइबर" क्या है?{#what-is-react-fiber}
 
-Fiber is the new reconciliation engine in React 16. Its main goal is to enable incremental rendering of the virtual DOM. [Read more](https://github.com/acdlite/react-fiber-architecture).
+फाइबर React 16 में नया सुलह इंजन है। इसका मुख्य लक्ष्य वास्तविक DOM के वृद्धिशील प्रतिपादन को सक्षम करना है। [अधिक पढ़ें](https://github.com/acdlite/react-fiber-architecture).
```
Comment (arshadkazmi42) [inline content/docs/faq-internals.md:23]: `सुलह` => `रीकोंसिलेशन`

**Review by arshadkazmi42 — CHANGES_REQUESTED** [review]
Great work.
Added review. Let's get these fixed and it will be good to go for next step of reviews.

For review process please read #23

File: content/docs/faq-internals.md (hunk 2 above)
Comment (bhatvikrant) [reply content/docs/faq-internals.md:13]: yea, I think `डेक्लेरेटिव` feels right

Comment (bhatvikrant) [reply content/docs/faq-internals.md:13]: Okay

**Comment by bhatvikrant** [comment]
@arshadkazmi42  I have addressed the requested changes 😇

**Comment by bhatvikrant** [comment]
Hey @arshadkazmi42 when can this be reviewed again?

**Comment by bhatvikrant** [comment]
Hey @arshadkazmi42, @saranshkataria I had made the requested changes, can this PR be moved forward whenever you guys have time :)

**Comment by arshadkazmi42** [comment]
We are moving our focus to new beta pages. So I will be closing this PR. Sorry about this.

But feel free to pick a new page from #168

**Pull request closed without merging**